src/v2/api/group/GroupService.ts

```diff
@@ -214,13 +226,17 @@ export class GroupService {
         set,
       }));
 
-      const role = await this.roleRepository.create({
+      await this.roleRepository.create({
         ...roles,
         grants: {
           create: grantList,
         },
+        groupRole: {
+          create: {
+            groupId,
+          },
+        },
       });
-      await this.groupRepository.addRole(role.id, groupId);
     }
   }
 
```

Comment:
delete this method cause' there is already an implementation in repository

Reply:
solved in ece85c0